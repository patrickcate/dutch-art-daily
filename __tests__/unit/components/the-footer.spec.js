import { shallowMount } from '@vue/test-utils'
import TheFooter from '@components/the-footer.vue'

describe('TheFooter Component', () => {
  it('is a Vue instance', () => {
    const wrapper = shallowMount(TheFooter)
    expect(wrapper.isVueInstance()).toBe(true)
  })

  it('renders correctly', () => {
    const wrapper = shallowMount(TheFooter)
    expect(wrapper).toMatchSnapshot()
  })
})
