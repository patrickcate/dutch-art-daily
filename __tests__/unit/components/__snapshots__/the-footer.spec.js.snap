// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheFooter Component renders correctly 1`] = `
<footer role="contentinfo" class="l-footer"><small class="l-footer__copyright">All artwork on this site is public domain and freely distributable. All
    uncited code and content is distributed under the MIT license. Feel free
    to
    <a href="https://github.com/patrickcate/dutch-art-daily" rel="noopener noreferrer external">
      view the code</a>.
  </small></footer>
`;
